data/pg/pucar/DocumentType.json

```diff
@@ -0,0 +1,160 @@
+{
+  "tenantId": "pg",
+  "moduleName": "pucar",
```

Comment:
These document types may need to go into common-masters as they may be used across modules. Please think through that. moduleName will become "common-masters" then.